src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -405,29 +423,48 @@ export class EditDataComponent extends GridParentComponent implements OnInit, On
 	}
 
 	private refreshGrid(): Thenable<void> {
+		//handle complete happens before refreshGrid is called for firstRender, something is not right.
 		return new Promise<void>((resolve, reject) => {
 			const self = this;
 			clearTimeout(self.refreshGridTimeoutHandle);
 			this.refreshGridTimeoutHandle = setTimeout(() => {
 				for (let i = 0; i < self.placeHolderDataSets.length; i++) {
-					self.placeHolderDataSets[i].dataRows = self.dataSet.dataRows;
-					self.placeHolderDataSets[i].resized.emit();
+					// TODO figure out why these values can now be null in some cases
```

Comment:
I assume the `TODO`'s will be removed by final PR review.  If not, please be sure to added your alias and date to the comment so we can track where\when the suggestion came from.